fs-common-bitproto/can.bitproto

```diff
@@ -1,23 +1,61 @@
 proto can
 
+/* =============== AMS ================= */	
+
+/* Current ID: 0x101 */
+message AMSReport {
+    /* Below are AIR Relay Feedback States */
+    bool AIR_POS = 6;   /* AIR Positive Relay State. 1 is closed, 0 is open */
+    bool PRECHARGE = 7;   /* Precharge Relay State. 1 is closed, 0 is open */
+    bool AIR_NEG = 8;   /* AIR Negative Relay State. 1 is closed, 0 is open */
+
+    /* Below are AMS's GPIO state */
+    bool AMS_AUX_GPIO1 = 9;   /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO2 = 10;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO3 = 11;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO4 = 12;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO5 = 13;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO6 = 14;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+    bool AMS_AUX_GPIO7 = 15;  /* AMS's AUX GPIO 1 state, reserved for future purpose. */
+
+    /* Below is about AMS firmware version*/
+    int8 AMS_firmware_version = 17;  /* Current AMS Firmware's version or equivalent info */
```

Comment:
If we wanna do this I recomend the first 4 char of a git commit and so please change this to a char[4] or other such data. This means that it auto sets the value depending on what git hash it is and so we can search through the commit history rather then each individual commit